core/src/main/groovy/com/novoda/gradle/release/ReleasePlugin.groovy

```diff
@@ -42,4 +43,8 @@ class ReleasePlugin implements Plugin<Project> {
         }
     }
 
+    private String getString(Project project, String propertyName, String defaultValue) {
+        project.hasProperty(propertyName) ? project.getProperty(propertyName) : defaultValue
```

Comment:
groovy is cool and all but I prefer the explicit `return ...` rather than implicitly it returns the last line of the function